lib/new_relic/agent.rb

```diff
@@ -117,6 +117,9 @@ class SerializationError < StandardError; end
     # been configured for
     AUTOMATIC_TRACER_MAX_ATTEMPTS = 60 # 60 = try about twice a second for 30 seconds
 
+    # Event types must consist of only alphanumeric characters, '_', ':', or ' '.
+    VALID_CUSTOM_EVENT_TYPE = /\A[\w: ]+\z/
```

Comment:
I took a look at the [agent spec (internal)](https://source.datanerd.us/agents/agent-specs/blob/main/Custom-Events.md) for custom event names, and I don't see anything about the alphanumeric/character limitations. Where did you find these?